src/App.js

```diff
@@ -10,18 +13,35 @@ class App extends Component {
     super();
 
     this.state = {
-      currentUser: 'Nathaniel'
+      currentUser: '',
+      id: ''
+      //maybe consider Nan lets do research
```

Comment:
Great call on this for not hard coding a user. Definitely will consider using Nan as a value for default id instead of an empty string.